example.yaml

```diff
@@ -0,0 +1,216 @@
+hero:
+  name: hero
+  hp:
+    base: 30
+    increase: 7
+  strength:
+    base: 12
+    increase: 3
+  speed:
+    base: 11
+    increase: 2
+enemies:
+  - name: rat
+    hp:
+      base: 10
+      increase: 3
+    strength:
+      base: 5
+      increase: 2
+    speed:
+      base: 16
+      increase: 2
+    distance:
+      Near: 12
+  - name: wolf
+    hp:
+      base: 15
+      increase: 3
+    strength:
+      base: 8
+      increase: 2
+    speed:
+      base: 12
+      increase: 2
+    distance:
+      Near: 9
```

Comment:
I think we shouldn't have to specify the distance like this (for one the number is not used). We should consider:

1. have a standalone field like `type: near` that gets translated to the group the enemies end up in
2. have sublists to group the enemies, e.g.
```yaml
enemies:
  -  near:
    - wolf: ...
    - rat: ...
  - middle:
    - ...
```
3. (if it's easier to implement) just skip the enemy grouping for now and have it hardcoded in the code.